app/src/main/java/com/framgia/oleo/screen/messages/MessageOptionViewModel.kt

```diff
@@ -0,0 +1,64 @@
+package com.framgia.oleo.screen.messages
+
+import android.app.Application
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModelProvider
+import com.framgia.oleo.base.BaseViewModel
+import com.framgia.oleo.data.source.UserRepository
+import com.framgia.oleo.data.source.model.FollowRequest
+import com.framgia.oleo.data.source.model.User
+import com.framgia.oleo.utils.Constant
+import com.google.firebase.database.DataSnapshot
+import com.google.firebase.database.DatabaseError
+import com.google.firebase.database.ValueEventListener
+import javax.inject.Inject
+
+class MessageOptionViewModel @Inject constructor(
+    private val application: Application,
+    private val userRepository: UserRepository
+) : BaseViewModel() {
+
+    val userFriend: MutableLiveData<User> by lazy {
+        MutableLiveData<User>()
+    }
+
+    val followRequest: MutableLiveData<FollowRequest> by lazy {
+        MutableLiveData<FollowRequest>()
+    }
+
+    fun getUserFriend(id: String) {
+        userRepository.getUserById(id, object : ValueEventListener {
+            override fun onCancelled(p0: DatabaseError) {
```

Comment:
trường hợp gặp lỗi phải show ra chứ thanh niên
```suggestion
            override fun onCancelled(error: DatabaseError) {
```